<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DBCViewer</class>
 <widget class="QMainWindow" name="DBCViewer">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>514</width>
    <height>523</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>100000</width>
    <height>10000</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>DBCViewer</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="2" column="0">
     <widget class="QTableWidget" name="tableWidget">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>1</width>
        <height>1</height>
       </size>
      </property>
      <property name="sizeIncrement">
       <size>
        <width>1</width>
        <height>1</height>
       </size>
      </property>
      <property name="baseSize">
       <size>
        <width>1</width>
        <height>1</height>
       </size>
      </property>
      <property name="editTriggers">
       <set>QAbstractItemView::NoEditTriggers</set>
      </property>
      <property name="showDropIndicator" stdset="0">
       <bool>false</bool>
      </property>
      <property name="dragDropOverwriteMode">
       <bool>false</bool>
      </property>
      <property name="selectionMode">
       <enum>QAbstractItemView::SingleSelection</enum>
      </property>
      <property name="selectionBehavior">
       <enum>QAbstractItemView::SelectRows</enum>
      </property>
      <property name="sortingEnabled">
       <bool>true</bool>
      </property>
      <property name="rowCount">
       <number>0</number>
      </property>
      <property name="columnCount">
       <number>1</number>
      </property>
      <attribute name="horizontalHeaderShowSortIndicator" stdset="0">
       <bool>true</bool>
      </attribute>
      <attribute name="verticalHeaderVisible">
       <bool>false</bool>
      </attribute>
      <attribute name="verticalHeaderHighlightSections">
       <bool>false</bool>
      </attribute>
      <column>
       <property name="text">
        <string>Id</string>
       </property>
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="background">
        <color>
         <red>11</red>
         <green>24</green>
         <blue>211</blue>
        </color>
       </property>
      </column>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QSplitter" name="splitter">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Ignored" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>496</width>
        <height>27</height>
       </size>
      </property>
      <property name="sizeIncrement">
       <size>
        <width>0</width>
        <height>0</height>
       </size>
      </property>
      <property name="baseSize">
       <size>
        <width>1</width>
        <height>1</height>
       </size>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QLineEdit" name="searchContent">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Ignored" vsizetype="Ignored">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>1</width>
         <height>1</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>400</width>
         <height>27</height>
        </size>
       </property>
       <property name="sizeIncrement">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
       <property name="baseSize">
        <size>
         <width>1</width>
         <height>1</height>
        </size>
       </property>
      </widget>
      <widget class="QComboBox" name="searchColumn">
       <property name="maximumSize">
        <size>
         <width>126</width>
         <height>27</height>
        </size>
       </property>
      </widget>
      <widget class="QPushButton" name="searchButton">
       <property name="maximumSize">
        <size>
         <width>143</width>
         <height>27</height>
        </size>
       </property>
       <property name="text">
        <string>Search</string>
       </property>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>514</width>
     <height>25</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="sizeIncrement">
    <size>
     <width>2</width>
     <height>2</height>
    </size>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <widget class="QMenu" name="menuOpen">
     <property name="title">
      <string>Open DBC</string>
     </property>
     <addaction name="actionAutomatic_Field_Detection"/>
     <addaction name="actionManual_Field_Setup"/>
    </widget>
    <widget class="QMenu" name="menuOpen_DB2">
     <property name="title">
      <string>Open DB2</string>
     </property>
     <addaction name="actionAutomatic_field_detection"/>
     <addaction name="actionManual_field_Setup"/>
    </widget>
    <addaction name="menuOpen"/>
    <addaction name="menuOpen_DB2"/>
    <addaction name="separator"/>
    <addaction name="actionReload"/>
    <addaction name="separator"/>
    <addaction name="actionExport_To_SQL"/>
    <addaction name="separator"/>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>View</string>
    </property>
    <widget class="QMenu" name="menuGet_Format_String">
     <property name="title">
      <string>Get Format String</string>
     </property>
     <addaction name="actionGet"/>
     <addaction name="actionSet"/>
    </widget>
    <addaction name="menuGet_Format_String"/>
    <addaction name="separator"/>
    <addaction name="actionFile_Statistics"/>
    <addaction name="separator"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuView"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="separator"/>
  </widget>
  <widget class="QStatusBar" name="statusBar">
   <property name="enabled">
    <bool>true</bool>
   </property>
  </widget>
  <action name="actionQuit">
   <property name="text">
    <string>Quit</string>
   </property>
  </action>
  <action name="actionGet">
   <property name="text">
    <string>Get</string>
   </property>
  </action>
  <action name="actionSet">
   <property name="text">
    <string>Set</string>
   </property>
  </action>
  <action name="actionReload">
   <property name="text">
    <string>Reload</string>
   </property>
  </action>
  <action name="actionAutomatic_Field_Detection">
   <property name="text">
    <string>Automatic field Detection</string>
   </property>
  </action>
  <action name="actionManual_Field_Setup">
   <property name="text">
    <string>Manual field Setup</string>
   </property>
  </action>
  <action name="actionExport_To_SQL">
   <property name="text">
    <string>Export To SQL</string>
   </property>
  </action>
  <action name="actionAutomatic_field_detection">
   <property name="text">
    <string>Automatic field detection</string>
   </property>
  </action>
  <action name="actionManual_field_Setup">
   <property name="text">
    <string>Manual field Setup</string>
   </property>
  </action>
  <action name="actionFile_Statistics">
   <property name="text">
    <string>File Statistics</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
